server/src/main/java/io/druid/client/cache/CacheMonitor.java

```diff
@@ -38,20 +39,30 @@ public CacheMonitor(
     this.cache = cache;
   }
```

Comment:
not a big deal but,
i meant, is anything actually using this constructor anymore. If not, then we can remove it.